components/nexus-ehcache/src/main/java/org/sonatype/sisu/ehcache/CacheManagerComponentImpl.java

```diff
@@ -48,8 +50,8 @@
   private CacheManager cacheManager;
 
   @Inject
-  public CacheManagerComponentImpl() throws IOException {
-    this(null);
+  public CacheManagerComponentImpl(final ApplicationDirectories directories) throws IOException {
+    this(directories.getAppDirectory("etc/ehcache.xml"));
```

Comment:
odd use of method, safer to use:

    new File(directories.getAppDirectory("etc"), "ehcache.xml")